tests/rss/test_rss.py

```diff
@@ -174,18 +243,23 @@ def test_m3gnet_rss(test_dir, memory_jobstore, clean_dir):
    
     assert len(output_filter) == 1
 
+    dir = Path('.')
+    path_to_job_files = list(dir.glob("job*"))
+    for path in path_to_job_files:
+        shutil.rmtree(path)
```

Comment:
This is not needed if you use `clean_dir`. Could you please fix this?

Reply:
Great! Will do.